data/webbrowser.yaml

```diff
@@ -1282,6 +1282,17 @@ sources:
 supported_os: [Darwin, Windows]
 urls: ['https://forensics.wiki/apple_safari']
 ---
+name: SafariCookies
+doc: Safari Cookies database.
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '%%users.homedir%%/Library/Cookies/*'
```

Comment:
any way this glob could be made more specific?

Reply:
sure, I was trying to make it resilient to any OSX changes, but I can't find any examples that aren't `Cookies.binarycookies`

Reply:
thanks, that's fine we can add additional paths in the future if needed